besu/src/main/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilder.java

```diff
@@ -266,13 +290,22 @@ public String build() {
       lines.add("Engine JWT: " + engineJwtFilePath);
     }
 
+    lines.add("Using " + txPoolImplementation + " transaction pool implementation");
+
     if (isHighSpec) {
       lines.add("Experimental high spec configuration enabled");
     }
 
-    lines.add("Using " + txPoolImplementation + " transaction pool implementation");
     lines.add("Using " + worldStateUpdateMode + " worldstate update mode");
 
+    if (trieLogRetentionThreshold > 0) {
```

Comment:
do we can use a second flag to disable the feature instead of 0 ?

Reply:
sure thing, any suggestions for a name (ideally that avoids "pruning")?

Reply:
Gone with `--Xbonsai-trie-log-pruning-enabled` against for now https://github.com/hyperledger/besu/pull/6026/commits/bde0bd27ca9dc347c6f212f471207503e6e630e1

Reply:
if we really want to remove pruning maybe this one ? `Xbonsai-trie-log-eviction-enabled`just a proposition not really pushing for that

Reply:
I will discuss name with Matt, but I think for now it doesn't matter while it's an experimental option. 
Despite what I'm said before, I'm currently thinking `pruning` might be OK after all because it accurately describes what it does and eventually I don't expect most users will need to know/worry about this feature.